### ISSUE TITLE
Grouping Guidelines and Outcomes by function and user/technology categories

### ISSUE THREAD

**Issue by rachaelbradley** [w3c/wcag3#82]
"To make it easier for content creators, Testers, etc to locate outcomes and understand the overall structure of WCAG 3.0, the guidelines and outcomes should be structured and grouped in a simplified, easy to reference document structure, similar to the POUR principles of previous versions of WCAG.  Additionally, each outcome should be logically grouped together by function.

For example, “Action required -The interface indicates when user input or action is required to proceed.” should be part of the Forms, Inputs, and Errors guideline, because Action Required includes required fields in forms.

Additionally, when structuring the guidelines and outcomes, it would be very helpful to Designers, Developers, and Testers if you could clearly categorize guidelines/outcomes  as applying to specific users/technologies.  For example, heading levels and Name, Role, Value, State  are semantic and apply more to non-visual techniques of reading and navigating such as screen readers.  Where as Color Contrast, Pointer Movements, and Focus Indicators are all related to Visual techniques for reading and navigating. While Unusual Words, Simpler Language, and Consistent Navigation can relate to people with Cognitive disabilities. And, Label In Name relates more closely with users of Speech To Text technologies.  Obviously, there are many success criteria/outcomes which can relate to multiple user/technology categories; this is why I think it would be best for the various user/technology categories to be assigned to each outcome as tags that can be searched and filtered.

As an Accessibility professional, I have been asked which specific guidelines relate to specific Assistive Technologies/Disability categories and it would be nice to be able to direct them to specific parts of the WCAG."

Issue submitted via public-agwg-comments email. Respond to email with update when addressed.

**Comment by detlevhfischer** [comment]
@rachaelbradley Is there a way to link to the mail in an archive and/or identify the commenter? Or is this comment intentionally anonymous once transferred to GitHub?

**Comment by rachaelbradley** [comment]
> @rachaelbradley Is there a way to link to the mail in an archive and/or identify the commenter? Or is this comment intentionally anonymous once transferred to GitHub?

Great point. Fixing that now.

**Comment by KevinP-NZ** [comment]
Sounds good as long as it is not too simplistic - guideline and outcomes will likely have one to many and many to many relationships as well as the ones that are clearly one-to one
